namespaces/live-1.cloud-platform.service.justice.gov.uk/offender-management-preprod/resources/rds-allocations-api.tf

```diff
@@ -0,0 +1,51 @@
+/*
+ * When using this module through the cloud-platform-environments, the following
+ * two variables are automatically supplied by the pipeline.
+ *
+ */
+
+variable "cluster_name" {}
+
+variable "cluster_state_bucket" {}
+
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "allocation-rds" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=4.0"
```

Comment:
please use the latest version of that module. Also, please use :
` db_instance_class      = "db.m4.large"`

example : https://github.com/ministryofjustice/cloud-platform-environments/blob/master/namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-visits-booking-production/resources/rds-prison-visits.tf

Reply:
Done